katas/metal-slug/rubiana/characters.js

```diff
@@ -0,0 +1,46 @@
+export class Character {
+    constructor(_character, _hole, _habilities, _health, _vehicle) {
+        this.character = _character;
+        this.hole = _hole;
+        this.habilities = _habilities;
+        this.health = _health;
+        this.vehicle = _vehicle;
+    }
+    //shoot a weapon, use a knife and throw granades
+    characters = [{
+        character: 'Soldier1',
+        hole: 'soldier',
+        habilities: 'habilities', //[shootWeapon(), useKnife(), throwGranades()],
+        health: 30,
+        vehicle: 'camel',
+    },
+    {
+        character: 'Enime1',
+        hole: 'enime',
+        habilities: 'habilities', //[shootWeapon(), useKnife(), throwGranades()],
+        health: 30,
+        vehicle: null,
+    },
+    ];
+
+    displayCharacters() {
+        return this.characters;
+    }
+}
+
+export class Soldier extends Character {
```

Comment:
This relationship makes sense, so I'd suggest to refactor your code based on this. You generalize soldier as a character, but you can add specific attributes/behavior to soldier class. Then, use your tests to show how soldiers can be created and what behavior you can expect from them